api/types/events/events.proto

```diff
@@ -2482,3 +2483,16 @@ message DatabaseSessionMalformedPacket {
     // Payload is the malformed packet payload.
     bytes Payload = 5 [ (gogoproto.jsontag) = "payload,omitempty" ];
 }
+
+// SessionRecordingView is emitted when a session is viewed in the web UI, allowing
+// session views to be included in the audit log
+message SessionRecordingView {
+    // Metadata is a common event metadata.
+    Metadata Metadata = 1
+        [ (gogoproto.nullable) = false, (gogoproto.embed) = true, (gogoproto.jsontag) = "" ];
+    // SessionID is the ID of the application session.
+    string SessionID = 2 [ (gogoproto.jsontag) = "sid" ];
+    // UserMetadata is a common user event metadata.
+    UserMetadata UserMetadata = 3
+        [ (gogoproto.nullable) = false, (gogoproto.embed) = true, (gogoproto.jsontag) = "" ];
+}
```

Comment:
Can we include the connection metadata (it would be nice to record the IP used to view the recording, for example)?

Reply:
I agree with Zac that this would definitely be nice to have, but we do need to ensure that the client IP is actually available to the auth server, and I'm sure the last time I checked there was a few situations where it wasn't.